bothub/api/grpc/organization/serializers.py

```diff
@@ -0,0 +1,120 @@
+from django.db.models import F
+from django_grpc_framework import proto_serializers
+from rest_framework import serializers
+
+from bothub.authentication.models import User
+from bothub.common.models import Organization, OrganizationAuthorization
+from bothub.protos import organization_pb2
+
+
+class SerializerUtils(object):
+    @classmethod
+    def get_object(cls, model, pk: int):
+        try:
+            return model.objects.get(pk=pk)
+        except model.DoesNotExist:
+            raise proto_serializers.ValidationError(
+                f"{model.__name__}: {pk} not found!"
+            )
+
+    @classmethod
+    def get_user_object(cls, model, email: str):
+        try:
+            return model.objects.get(email=email)
+        except model.DoesNotExist:
+            raise proto_serializers.ValidationError(
+                f"{model.__name__}: {email} not found!"
+            )
+
+
+class OrgProtoSerializer(proto_serializers.ModelProtoSerializer):
+
+    users = serializers.SerializerMethodField()
+
+    def get_users(self, org: Organization):
+        return list(
```

Comment:
this queryset could not be in the Organization model? If you need it in another code, will you duplicate the queryset?